app/Actions/Place/GetPlaceCollection/GetPlaceCollectionPresenter.php

```diff
@@ -8,8 +8,36 @@ public static function present(GetPlaceCollectionResponse $placeResponse): array
     {
         $placesArray = [];
 
-        foreach ($placeResponse->getPlaceCollection() as $place) {
-            $placesArray[] = $place;
+        foreach ($placeResponse->getPlaceCollection() as $key => $place) {
+            $placesArray[$key]['id'] = $place->id;
```

Comment:
Move transforming to a separate method like

`foreach ($places as $place) {  $new[] = $this->presentAsArray($place)   }`